server/v2/cometbft/abci_test.go

```
@@ -15,6 +16,7 @@ import (
 	abciproto "github.com/cometbft/cometbft/api/cometbft/abci/v1"
 	v1 "github.com/cometbft/cometbft/api/cometbft/types/v1"
 	"github.com/cosmos/gogoproto/proto"
+	gogoproto "github.com/cosmos/gogoproto/proto"
```

golangci-lint (1.62.2)

ST1019(related information): other import of "github.com/cosmos/gogoproto/proto"

(stylecheck)